BossMod/BossModule/AIHints.cs

```diff
@@ -36,6 +36,7 @@ public enum SpecialMode
 
     public WPos Center;
     public ArenaBounds Bounds = DefaultBounds;
+    public WaypointManager WaypointManager { get; private set; } = new WaypointManager();
```

Comment:
Hints are supposed to be stateless (cleared and filled from scratch every frame), stuff like replay rewind etc relies on that. I also don't really believe randomness has any place here - if anything, it will complicate debugging, and I don't really see a benefit (why would someone care about bot-like behaviour when botting?..)